studio/src/pages/_error.jsx

```diff
@@ -0,0 +1,17 @@
+import * as Sentry from "@sentry/nextjs";
+import Error from "next/error";
+
+const CustomErrorComponent = (props) => {
```

Comment:
What we definitly need to do is to add the orgId, userId  to the sentry context after authentication. You get the authenticated user in `app-provider.tsx`

See: https://docs.sentry.io/platforms/javascript/session-replay/configuration/#identifying-users